### ISSUE TITLE
please guide for installation step and step for plyr

### ISSUE THREAD

**Issue by manuelsongokuh** [sampotts/plyr#2539]
im look for , look for, search search, i dont understand as correct information for installation in code, hmtl, js, css..
there is confusion..

plyr.3.7.2.zip has all files, i open files dont understand nothing what i need install in my server ? all files or choose file.. i dont see nothing step step guide..
confusion 

i check codepen has  2 o 3 files.. instead plyr.3.7.2.zip has million files..what i will do..

i dont find understand how change language for UI , is confusion..

please help me file: html, css, js, all works simple for installation.. version 3.7.2 i dont find file..

**Comment by manuelsongokuh** [comment]
see example:
correct from site
![immagine](https://user-images.githubusercontent.com/12126929/194404353-6419ce7b-4c00-4272-bc7a-f4c1902dbc9f.png)

instead: file local test preview is bad : 

![immagine](https://user-images.githubusercontent.com/12126929/194404442-8698462a-729e-480f-97c4-3307d346be82.png)

**Comment by manuelsongokuh** [comment]
nothing reply here...?

**Comment by gabriel-tandil** [comment]
The same thing happens to me,
I guess it shouldn't be that difficult and that's why it's not explained, but I think it would help to have a detailed user's guide.

**Comment by manuelsongokuh** [comment]
@gabriel-tandil yes , me too, same think, so that is needed user's guide..this plyr is beautifull but need helped..i try waiting..

**Comment by gabriel-tandil** [comment]
@manuelsongokuh I searched in my favorite web search engine and found [this](https://dyclassroom.com/reference-javascript/player-for-website-video-and-audio-using-plyr)
I'll see if it works later

**Comment by manuelsongokuh** [comment]
i did this side but version is old:
<script src="https://cdn.plyr.io/2.0.15/plyr.js"></script>

but i need more example icon svg, player large.. etc..

**Comment by davidlowry** [comment]
Read the README - https://github.com/sampotts/plyr 

Include script in header:
`<script src="https://cdn.plyr.io/3.7.3/plyr.js"></script>`

And the css:
`<link rel="stylesheet" href="https://cdn.plyr.io/3.7.3/plyr.css" />`

Make a dom:
`<video id="player" playsinline controls data-poster="/path/to/poster.jpg">
  <source src="/path/to/video.mp4" type="video/mp4" />
  <source src="/path/to/video.webm" type="video/webm" />

  
  <track kind="captions" label="English captions" src="/path/to/captions.vtt" srclang="en" default />
</video>`

Init the player
`<script>
  const player = new Plyr('#player');
</script>`
